### ISSUE TITLE
1989 Issue

### ISSUE THREAD

**Issue by regan008** [regan008/MGG-LC-Data#7]

**Comment by cblevins** [comment]
- [ ] Roughly 2 pages of entries for San Fran are missing

**Comment by lsmith13** [comment]
Clean-up completed, ready for geocoding. @regan008 @cblevins @heinz-am

**Comment by heinz-am** [comment]
Are we still missing those 2 pages? According to my records, this 1989 issue came from a scan via Clemson. @regan008 , sorry to put something else on your plate -- are you able to follow up?

**Comment by regan008** [comment]
@heinz-am You are totally right - I had missed that comment. The pages are all there in the scan, but I think they were accidentally missed during transcription. Is this something @lsmith13 could work on for us? It looks like we got the first 3 entries on pg 180 (guide pg, not pdf) but we're missing the remainder of the SF entries. We just need to go back and add all of the SF entries, and it looks like we're missing the better part of 10 pages.

**Comment by lsmith13** [comment]
No problem @heinz-am  @regan008, I'll get started on this right now.

**Comment by lsmith13** [comment]
@heinz-am @regan008 completed adding the missing pages
